package.json

```diff
@@ -6,60 +6,59 @@
   "scripts": {
     "start": "node $NODE_APP_OPTIONS app.js",
     "lint": "node_modules/.bin/eslint .",
-    "format": "node_modules/.bin/prettier-eslint '**/*.js' --list-different",
-    "format:fix": "node_modules/.bin/prettier-eslint '**/*.js' --write",
+    "format": "node_modules/.bin/prettier-eslint $PWD'/**/*.js' --list-different",
+    "format:fix": "node_modules/.bin/prettier-eslint $PWD'/**/*.js' --write",
     "test:unit": "npm run test:unit:_run -- --grep=$MOCHA_GREP",
     "test:acceptance": "npm run test:acceptance:_run -- --grep=$MOCHA_GREP",
     "test:unit:_run": "mocha --recursive --reporter spec $@ test/unit/js",
-    "test:acceptance:_run": "mocha --recursive --reporter spec --timeout 15000 --exit $@ test/acceptance/js",
+    "test:acceptance:_run": "mocha --exit --recursive --reporter spec --timeout 15000 $@ test/acceptance/js",
     "nodemon": "nodemon --config nodemon.json"
   },
   "dependencies": {
-    "async": "^3.1.0",
-    "bcrypt": "^3.0.6",
+    "async": "^3.2.0",
+    "bcrypt": "^4.0.1",
```

Comment:
Major version update, but the changelog looks OK. :+1:

Reply:
Note: if we were using read-only, which at the moment we aren't, I think the main question is whether it can read hashes generated by the version of bcrypt in web. They are usually pretty good about forward compatibility, so I would be surprised if they're not, but that's the main requirement here I think.

Reply:
I'd be really surprised if that becomes a problem. Do we have read-only in staging for testing?

Reply:
I think it is probably broken, because we removed the database it was using CC @henryoswald 

In any case, the current plan is to delete read-only entirely for the time being, so probably we can live with it --- we'll have to do that testing we bring it back.

Reply:
I was wondering, should we merge now (to prevent any activity in `master` to break the PR) or should we wait until we bring it back?

Reply:
I think we should go ahead and merge this; we don't have to deploy it, since it's quite broken now anyway.